src/hooks/useCds/index.js

```diff
@@ -144,6 +147,18 @@ const applyCds = async function(patientData, setOutput, setIsLoadingCdsData, isT
       decisionAids = { errors };
     }
 
+    // replace with actual logging data when ready from CQL 
+    // output otherResources as a temporary stand-in
+    console.timeEnd('Apply CDS');
+
+    if (LOGGER_ENABLED){
+      logMsg({
+        timeRequestSent: new Date(),
+        patientReference: patientReference,
+        payload: otherResources
```

Comment:
Are we planning to log the `otherResources` object as a part of this PR, and then update what we are actually logigng in the future? Or will this PR directly be updated with the correct payload from the CQL?

Reply:
I believe it will be a separate PR. We may prioritize delivering an early working version of the loggig service so that UMMC get the tool be aligned with their IT env, meanwhile we're working on CQL part in parallel.